pinot-connectors/pinot-flink-connector/src/test/resources/fixtures/pinotTableSchema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "schemaName": "demand",
+  "dimensionFieldSpecs": [
+    {
+      "name": "demand_uuid",
+      "singleValueField": true,
+      "dataType": "STRING",
+      "virtualColumnProvider": null,
+      "defaultNullValue": "null",
+      "transformFunction": null,
+      "maxLength" : 4096
+    },
+    {
+      "name": "geofence",
+      "singleValueField": true,
+      "dataType": "STRING",
+      "virtualColumnProvider": null,
+      "defaultNullValue": "null",
+      "transformFunction": null
+    }
+  ],
+  "metricFieldSpecs": [
+    {
+      "dataType": "DOUBLE",
+      "singleValueField": true,
+      "fieldSize": 8,
+      "derivedMetricType": null,
+      "name": "surge_multiplier",
+      "virtualColumnProvider": null,
+      "defaultNullValue": 0.0,
+      "transformFunction": null
+    }
+  ],
+  "dateTimeFieldSpecs": [
+  ],
+  "timeFieldSpec": {
```

Comment:
nit: use the dateTimeFieldSpec. often our oss users look at the quickstarts and copy the configs. we don't want anyone to be using timeFieldSpec anymore